package-lock out of date

<!--
  For __feature requests__, please ignore this template (you can delete it)
  Instead please include a description of the desired feature, and a brief
  description of the scenario in which you intend to use it
-->
We need to update our lockfile - for some reason it isn't currently up to date, which breaks ci 😨 

### Environment

<!--
  Please fill in all the relevant fields by running these commands in terminal.
-->

4. `node -v`: v10.9.0
5. `npm -v`: v6.2.0

Then, specify:

<!-- Which operating system are you using? Specify macOS, Windows, or Linux, along with specific release versions -->
- Development Operating System: ubuntu trusty

### Steps to Reproduce

<!--
  How would you describe your issue to someone who doesn’t know you or your project?
  Try to write a sequence of steps that anybody can repeat to see the issue.
  Be specific! If the bug cannot be reproduced, your issue may be closed.
-->

(Write your steps here:)

1. Attempt to execute a ci build (`npm ci`)
2. Notice failure due to lockfile
3. Wish there was no failure 😄 

### Expected Behavior

<!--
  How did you expect your project to behave?
  It’s fine if you’re not sure your understanding is correct.
  Just write down what you thought would happen.
-->

A clean build should be produced.

### Actual Behavior

<!--
  Did something go wrong?
  Is something broken, or not behaving as you expected?
  Describe this section in detail, and attach screenshots if possible.
  Don't just say "it doesn't work"!
-->

```
npm ERR! cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
npm ERR! 
npm ERR! 
npm ERR! Missing: eslint@^4.19.1
npm ERR! Missing: eslint-config-standard@^11.0.0
npm ERR! Missing: eslint-plugin-import@^2.11.0
npm ERR! Missing: eslint-plugin-node@^6.0.1
npm ERR! Missing: eslint-plugin-promise@^3.7.0
npm ERR! Missing: eslint-plugin-standard@^3.0.1
npm ERR! 
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2018-08-22T06_15_34_561Z-debug.log
The command "eval npm ci  " failed. Retrying, 2 of 3.
```

--- a/package.json
+++ b/package.json
@@ -14,12 +14,12 @@
   "author": "Ben Greenier",
   "license": "MIT",
   "devDependencies": {
-    "eslint": "^4.19.1",
-    "eslint-config-standard": "^11.0.0",
-    "eslint-plugin-import": "^2.11.0",
-    "eslint-plugin-node": "^6.0.1",
-    "eslint-plugin-promise": "^3.7.0",
-    "eslint-plugin-standard": "^3.0.1",
-    "mocha": "^5.0.0"
+    "eslint": "^5.4.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-node": "^7.0.1",
+    "eslint-plugin-promise": "^4.0.0",
+    "eslint-plugin-standard": "^4.0.0",
+    "mocha": "^5.2.0"
   }
 }

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
     "platform",
     "chaos"
   ],
-  "author": "Ben Greenier",
+  "author": "microsoft",
   "license": "MIT",
   "devDependencies": {
     "eslint": "^5.4.0",

